code/_planes+layers.dm

```diff
@@ -75,34 +75,36 @@ What is the naming convention for planes or layers?
 
 #define ABOVE_PLATING_PLANE		-20
 
-	#define CATWALK_LAYER			0
-	#define DISPOSALS_PIPE_LAYER	1
-	#define LATTICE_LAYER			2
-	#define PIPE_LAYER				3
-	#define WIRE_LAYER				4
-	#define VENT_BEZEL_LAYER		5
-	#define WIRE_TERMINAL_LAYER		6
+	#define HOLOMAP_LAYER			1 // NOTE: ENSURE this is equal to the one at ABOVE_TURF_PLANE!
```

Comment:
Well yeah but comic is saying here to not use an identical define name twice and instead have two distinct names one of which defines the other? Why would you make the same define twice anyway.